.github/workflows/ci_build.yml

```diff
@@ -0,0 +1,56 @@
+# preforms CI builds using compose.py from
+# https://github.com/CleverRaven/Cataclysm-DDA
+#
+# This action is triggerd by any PR against the master branch as well
+# as on any push to the master branch itself
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+name: CI Build
+
+jobs:
+  build:
+    name: CI Build
+    runs-on: ubuntu-latest
+    container: alpine:latest
```

Comment:
why is the container used?

Reply:
iirc, I used that because the old ubuntu version github provided didn't have either a recent enough libvips or any libvips in its repos.